doc/api/examples/modelRepositories2.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You could also reference `modelRepository2.json` in `modelRepositories.json` to create an example where multiple model Repos are shown